wallet/chain/p/builder/builder.go

```diff
@@ -374,7 +374,7 @@ func (b *builder) NewBaseTx(
 	options ...common.Option,
 ) (*txs.BaseTx, error) {
 	toBurn := map[ids.ID]uint64{
-		b.context.AVAXAssetID: b.context.StaticFeeConfig.TxFee,
+		b.context.AVAXAssetID: b.context.TxFee,
```

Comment:
I think every usage of `TxFee` can be replaced with `0` (or if they are entries in maps we can just remove it, like here).
```suggestion
```